app/profile/manageTask/page.tsx

```diff
@@ -14,10 +14,10 @@ import Task from "@/components/Task/Task";
 import Loader from "@/components/Loader/Loader";
 
 export default function ManageTask() {
-  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [isLoading, setIsLoading] = useState(true);
   const [tasksData, setTasksData] = useState<DataType[] | []>([]);
-  const [currentCategory, setCurrentCategory] = useState<string>("all");
-  const [currentImportance, setCurrentImportance] = useState<string>("all");
+  const [currentCategory, setCurrentCategory] = useState("all");
+  const [currentImportance, setCurrentImportance] = useState("all");

```

Comment:
Kategorie i to Importance przeniósłbym do enuma i określił tam wartości, nie widzę tutaj jakie są inne ale np:
ALL = "all",
COSTAM = "costam";

I wtedy tutaj na inicjalnej wartosci ustalasz CATEGORIES.ALL, a w miejscu gdzie uruchamiasz set'a to tez ustawiasz CATEGORIES.COSTAM np. Jak najmniej stringow, jak najwiecej stałych, enumów szczególnie gdy mamy jakis zbiór mozliwych wartosci